src/uuid/builder/uuidbuilder.rs

```diff
@@ -128,42 +123,40 @@ impl UUIDFactory for UUIDv6Factory {
 /// | var        | MUST be the The 2 bit variant defined by Section 4.1 of RFC
 /// | rand_b     | MUST 62 bits random number that is generated at initialization time of the uE only and reused otherwise |
 
-pub struct UUIDv8Factory {
+pub struct UUIDv8Builder {
     msb: Arc<Mutex<u64>>,
```

Comment:
Could an `AtomicU64` be used instead of an `Arc<Mutext<u64>>`?

Reply:
done